internal/backends/winit/renderer/femtovg/itemrenderer.rs

```diff
@@ -1175,14 +1175,19 @@ impl<'a> GLItemRenderer<'a> {
                 let image = source_property.get();
                 let image_inner: &ImageInner = (&image).into();
 
-                let target_size_for_scalable_source = image_inner.is_svg().then(|| {
-                    // get the scale factor as a property again, to ensure the cache is invalidated when the scale factor changes
+                let target_size_for_scalable_source = if image_inner.is_svg() {
+                    let image_size = image.size();
```

Comment:
Ah yes, a clash between our sync Image API and the async image loading in the wasm interpreter :(